doc/ruleSets/PER-CS2.0Risky.rst

```diff
@@ -2,7 +2,15 @@
 Rule set ``@PER-CS2.0:risky``
 =============================
 
-Rules that follow `PER Coding Style 2.0 <https://www.php-fig.org/per/coding-style/>`_. This set contains rules that are risky.
+Rules that follow `PER Coding Style 2.0 <https://www.php-fig.org/per/coding-style/>`_.
+
+Warning
+-------
+
+This set contains rules that are risky
```

Comment:
I like this warning. pR description is not mentioning this change. Maybe we should split PR into 2 ?

Reply:
This message was already here, but glued after the ruleset's description. I've introduced "Warnings" section to be in sync with fixers' docs, so this warning was just moved to the new section. Splitting it to 2 PRs does not make much sense IMHO.